.github/workflows/_finalize.yaml

```diff
@@ -8,22 +8,121 @@ on:
         description: Update the landing page badges with run outcomes
         default: false
         required: false
+      # upload badge
```

Comment:
This comment is vague. Please extend it to make it more informative. Additionally, for the workflow inputs/outputs section, the `description` field is the perfect place for documentation.